src/components/ComboButton/ComboButton.js

```diff
@@ -169,6 +170,28 @@ ComboButton.propTypes = {
   /** @type {string} Overflow menu direction. */
   direction: PropTypes.oneOf([TooltipDirection.TOP, TooltipDirection.BOTTOM]),
 
+  /**
+   * The adjustment in position applied to the floating menu.
+   */
+  menuOffset: PropTypes.oneOfType([
+    PropTypes.shape({
+      top: PropTypes.number,
+      left: PropTypes.number,
```

Comment:
Could these values also be strings, like `'auto'`?

Reply:
Good point! Will update that 👍